doc/panes/editor.rst

```diff
@@ -29,32 +31,275 @@ The Editor pane consists of the following areas:
+* The :doc:`plots` pane renders the figures and images created.
+* The :doc:`profiler` helps you optimize your code's run time.
+
+
+Run file
+~~~~~~~~
+
+Run an entire Editor file using the :menuselection:`Run --> Run` menu item, the :guilabel:`Run file` toolbar button or the :kbd:`F5` key.
+Use :menuselection:`Run --> Re-Run last script` to re-run the most recent file executed with the above.
+
+
+Run line/selection
+~~~~~~~~~~~~~~~~~~
+
+You can execute the current line—or multiple selected lines—using the :guilabel:`Run selection or current line` option from the toolbar or the :menuselection:`Run` menu, as well as with the :kbd:`F9` key.
+After running the current line, the cursor automatically advances to the next, so you can step through your code line by line.
+Unlike :guilabel:`Run file`, the executed lines are shown in the :doc:`ipythonconsole`.
+
+
+Code cells
+~~~~~~~~~~
+
+To run a cell, place your cursor inside it and use the :menuselection:`Run --> Run cell` menu item, the :guilabel:`Run current cell` toolbar button or the :kbd:`Ctrl-Enter` / :kbd:`Cmd-Return` keyboard shortcut.
+Use :guilabel:`Run cell and advance` in the :guilabel:`Run` menu/toolbar or :kbd:`Shift-Enter` to jump to the next cell after running, useful for stepping through cells quickly.
+
+
+Run configuration
+~~~~~~~~~~~~~~~~~
+
+You can use the :guilabel:`Run configuration per file` dialog to set each file's working directory, console mode (current, dedicated or external), CLI arguments, execution options (clear all variables, run in an existing/empty namespace, debug on error), and more.
+To access it, click :menuselection:`Run --> Configuration per file...` or press :kbd:`Ctrl-F6` / :kbd:`Cmd-F6`.
+
+
+
+===============
+Code navigation
+===============
+
+Find and replace
+~~~~~~~~~~~~~~~~
+
+To search for text in the current file, use :menuselection:`Search --> Find text` or :kbd:`Ctrl-F` / :kbd:`Cmd-F`, and to replace it, use :menuselection:`Search --> Replace text` or :kbd:`Ctrl-R` / :kbd:`Cmd-R`.
+Typing your search string in the resulting panel below the Editor highlights each result and counts the total.
+Navigate between matches with :guilabel:`Find Previous` and :guilabel:`Find Next` in the find/replace panel and the :guilabel:`Search` menu, or use the :kbd:`F2` and :kbd:`F3` keys.
+Use the :guilabel:`.*` button to process search text as a `regular expression <https://docs.python.org/3/library/re.html>`_, :guilabel:`Aa` to treat it as case-sensitive and :guilabel:`[–]` to only match whole words (e.g. for ``data``, match ``data()`` but not ``dataframe``).
+
+.. image:: /images/editor/editor-find-replace-panel.png
+   :alt: Spyder's Editor pane, showing the find and replace panel
+
+
+Go to line
+~~~~~~~~~~
+
+The :guilabel:`Go to line` dialog allows jumping to a specific line in the active file.
+Open it with :menuselection:`Search --> Go to line` or :kbd:`Ctrl-L` / :kbd:`Cmd-L`, type the line you want and press :kbd:`Enter` (or click :guilabel:`OK`).
```

Comment:
```suggestion
Open it with :menuselection:`Search --> Go to line` or :kbd:`Ctrl-L` / :kbd:`Cmd-L`, type the line number you want to scroll to and press :kbd:`Enter` (or click :guilabel:`OK`).
```

Slight tweak to use the more precise and less repetitive "scroll" instead of go, otherwise :+1: